### DISCUSSION TITLE
New maintainers - thanks @manmay-nakhashi!

### DISCUSSION THREAD

**Discussion by neonbjb** [neonbjb/tortoise-tts#513] [Announcements]
Just a quick announcement - @manmay-nakhashi graciously volunteered to help maintain Tortoise. He's done a great job building out speed enhancements in https://github.com/manmay-nakhashi/tortoise-tts-fastest. Thank you very much Manmay!

**Comment by 152334H** [comment]
cool

**Comment by reihanh** [comment]
That's nice

**Comment by LucidFir** [comment]
Should we be using that then instead of your one? Having the webui might be nice, there's also the other version with the better webui...

**Reply by manmay-nakhashi** [reply]
This is a main repo everyone builds on top of this repo.

**Reply by jjjota23** [reply]
How can we train in Portuguese? I have a source with more than 50,000 hours of audio, clean and of very high sound quality.

**Reply by 152334H** [reply]
where
